Cargo.lock

```diff
@@ -12,7 +12,7 @@ dependencies = [
  "arc-swap",
```

Comment:
The lock file and deny.toml seem to contain a lot of changes that aren't in the Cargo.toml in this PR. But they should have disappeared when PR #7255 merged.

Can you force-push or rebase this PR to make the diff smaller?

Reply:
> But they should have disappeared when PR https://github.com/ZcashFoundation/zebra/pull/7255 merged.

Yeah, that's weird. 

> Can you force-push or rebase this PR to make the diff smaller?

Done. It still shows the commit but the diff looks right now.